src/store/global/global.thunks.ts

```diff
@@ -378,7 +370,11 @@ function checkPendingDelayedWithdrawals() {
         const { hermezEthereumAddress } = wallet;
         const provider = Providers.getProvider();
         const accountEthBalance = BigInt(
-          await provider.getBalance(Addresses.getEthereumAddress(hermezEthereumAddress))
+          await provider.getBalance(Addresses.getEthereumAddress(hermezEthereumAddress)).then(
+            // ToDo: Why is this conversion now required? Is it safe as it's?
```

Comment:
This has been done to work with native `BigInt` instead of the `ethers` ones. This should be removed in #645 by @adrifdez.

Reply:
Yeah! This has been removed!